docs/rules/no-prototype-builtins.md

```diff
@@ -32,4 +36,4 @@ var barIsEnumerable = {}.propertyIsEnumerable.call(foo, "bar");
 
 ## When Not To Use It
 
-You may want to turn this rule off if you will never use an object that shadows an `Object.prototype` method or which does not inherit from `Object.prototype`.
+You may want to turn this rule off if your code never handles user input, and you will never use an object that shadows an `Object.prototype` method or which does not inherit from `Object.prototype`.
```

Comment:
> if your code never handles user input,

+ imagine plain strings -- it's safe unless using something like `JSON.parse()`. 
+ should also include imported object from third-party libs.

consider something like `untrusted object`?